zcash_client_sqlite/src/wallet/common.rs

```diff
@@ -289,7 +319,7 @@ where
            SELECT {table_prefix}_received_note_id
```

Comment:
The documentation just above this statement (specifically the text after the semicolon) is incorrect after this PR.